react-file-uploads/server/index.js

```diff
@@ -0,0 +1,66 @@
+require('dotenv/config');
+const path = require('path');
+const pg = require('pg');
+const express = require('express');
+const ClientError = require('./client-error');
+const errorMiddleware = require('./error-middleware');
+const uploadsMiddleware = require('./uploads-middleware');
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+const publicPath = path.join(__dirname, 'public');
+
+if (process.env.NODE_ENV === 'development') {
+  app.use(require('./dev-middleware')(publicPath));
+}
+
+app.use(express.static(publicPath));
+app.use(express.json());
+
+app.post('/api/uploads', uploadsMiddleware, (req, res, next) => {
+  const { caption } = req.body;
+  if (!caption) {
+    throw new ClientError(400, 'caption is a required field');
+  }
+  /**
+   * - create a url for the image by combining '/images' with req.file.filename
+   * - insert the "caption" and "url" into the "images" table
+   * - respond with the inserted row data
+   * - catch any errors
+   */
+  const url = `/images${req.file.filename}`;
```

Comment:
you'll want to include a `/` after `/images` so that the url works when you go to `localhost:3000/images/<filename>` in the browser